projects/aca-about/src/lib/about.component.ts

```diff
@@ -23,12 +23,12 @@
  * along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
  */
 
-import { Component, Inject, OnInit } from '@angular/core';
+import { Component, Inject, OnInit, Optional } from '@angular/core';
 import { DEV_MODE_TOKEN } from './dev-mode.tokens';
-import pkg from 'package.json';
```

Comment:
Cannot build library if file is imported from outside of the library root folder, therefore provided it using service